src/child-view-container.js

```diff
@@ -112,11 +122,8 @@ _.extend(Container.prototype, {
   },
 
   // Find a view by the `cid` of the model that was attached to it.
-  // Uses the model's `cid` to find the view `cid` and
-  // retrieve the view using it.
   findByModelCid(modelCid) {
-    const viewCid = this._indexByModel[modelCid];
-    return this.findByCid(viewCid);
+    return this._indexByModel[modelCid];
```

Comment:
somewhat unrelated, but there was no reason to store the view cid instead of the view.